docs/usermanual.md

```diff
@@ -313,13 +313,13 @@ result to another topic.
 The _commands topic_, where the client will be subscribed has the following format:
 
 ```text
-<apiKey>/<deviceId>/cmd
+/ul/<apiKey>/<deviceId>/cmd
```

Comment:
Usage prefix should be reviewed in documentation after recent discussions in skype... Originally we thought in using prefix for commands but now we don't use it but not sure if documentation changes have been updated accordinly.

I guess it may affect to .../cmd topic, .../cmdexe topic or both (not sure...).

Reply:
Updated in https://github.com/telefonicaid/iotagent-ul/pull/401/commits/34b22715e49db3223cb98de2ae189bd761e294e4
All the topics subscribed by the agent (to send measures, to configuration command retrieval or to get result of a command) are prefixed with the agent procotol (/ul)
All topis published by the agent (to send a comand or to send configuration information) to a device are not prefixed by the protocol, in this case '/ul',